src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -21,7 +22,10 @@ import org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBui
 @RestController
 @Api(tags = ["klage-oppgave-api"])
 @ProtectedWithClaims(issuer = ISSUER_AAD)
-class OppgaveController(val oppgaveService: OppgaveService) {
+class OppgaveController(
+    private val oppgaveService: OppgaveService,
+    private val innloggetSaksbehandlerRepository: InnloggetSaksbehandlerRepository
```

Comment:
Å gå rett på et `repository` fra controller føles kanskje litt rart? Men kan jo være at du bare tenker debug her da.

Reply:
Dette var bare for debuggingens skyld, ja.